code/modules/admin/verbs/selectequipment.dm

```diff
@@ -0,0 +1,197 @@
+/client/proc/cmd_select_equipment(mob/M in GLOB.mob_list)
```

Comment:
https://github.com/tgstation/tgstation/blob/master/.github/CONTRIBUTING.md#use-descriptive-and-obvious-names